test/backend_test.in.cpp

```diff
@@ -441,6 +441,41 @@ NGRAPH_TEST(${BACKEND_NAME}, concat_matrix_int64)
               read_vector<int64_t>(result));
 }
 
+// Add this test to cover paramter space overflow:
+// cuda kernel parameter space have limit, if there is large number of parameters,
+// there will be overflow for parameter space.
+NGRAPH_TEST(${BACKEND_NAME}, concat_vector_large)
+{
+    Shape shape_a{1};
+    NodeVector inputs;
+    op::ParameterVector inputs_param;
+    for (uint32_t i = 0; i < 999; i++)
```

Comment:
This only tests case when number of inputs is > your split point. Would be good to test exactly same number as your split point (concat_vector test already tests < your split point). Since this is reference case driven, you could do this as a parameterized test (I added support for this and can describe mechanism) and generate 2 or more variants.